runtime/gc_base/GenerationalAccessBarrierComponent.hpp

```diff
@@ -53,7 +53,7 @@ class MM_GenerationalAccessBarrierComponent : public MM_BaseNonVirtual
 	void tearDown(MM_EnvironmentBase *env);
 
 	void postObjectStore(J9VMThread *vmThread, J9Object *dstObject, J9Object *srcObject);
-	void preBatchObjectStore(J9VMThread *vmThread, J9Object *dstObject);
+	void postBatchObjectStore(J9VMThread *vmThread, J9Object *dstObject);
```

Comment:
I believe (although not sure) that generational barrier could be either post or pre barrier (in which case these two should not have any pre or post prefix).
We should come to it eventually when doing generational+SATB, when we should try concurrent and generational together in one pre barrier call (rather than having both pre barrier called just for concurrent part and post barrier called for generational part).

Reply:
Removed prefix altogether from them - makes sense not to have it.